src/src/assets/scss/mixins.scss

```diff
@@ -0,0 +1,237 @@
+@import './variables.scss';
+@import './functions.scss';
+
+/// Mixin to prefix a property
+/// @author Hugo Giraudel
+/// @param {String} $property - Property name
+/// @param {*} $value - Property value
+/// usage @include vendor-prefix('box-sizing', 'border-box');
+@mixin prefix($property, $value, $important: false) {
+    @each $prefix in ('webkit', 'moz', 'ms', 'o') {
+        #{'-' + $prefix + '-' + $property}: $value;
+    }
+
+    // Output standard non-prefixed declaration
+    @if $important {
+        #{$property}: $value !important;
+    }
+
+    @else {
+        #{$property}: $value;
+    }
+}
+
+@mixin flexbox {
+    display: flex;
+    display: -moz-box;
+    display: -webkit-flexbox;
+    display: -ms-flexbox;
+    display: -webkit-flex;
+    display: -moz-flex;
+    min-height: 1px;
+    min-width: 1px;
+}
+
+@mixin container {
+    width: $grid-xxl;
+    left: 0;
+    right: 0;
+    margin-left: auto !important;
+    margin-right: auto !important;
```

Comment:
Try to avoid !important as much as possible. Since this is a mixin, this should fixed in another way, or there will be complicated CSS fixes in the future.